### DISCUSSION TITLE
Dual or Multiple Birdseye Configs

### DISCUSSION THREAD

**Discussion by Icantpickaname** [blakeblackshear/frigate#9471] [Ideas]
This may not be doable, or not called for enough to warrant attention or implementation, but it would be great for my purposes, to have varying instances of Birdseye. I love this feature in particular. I have ten cameras in and around my home at the moment, with plans to expand. I have two front cameras, a single landing and a single back camera enabled for Birdseye currently. I'd like to have a Birdseye instance or quad channel mode, for each area or specific areas, where possible. I know there are cards for HA which work, but they come with a delay that the frigate webapp/web access does not have and whilst I also love HA, I like to separate instances/entities whenever possible. I rely on my cameras for security to some degree, but it's mainly to check on my cats and to see when a delivery arrives in the road or is at my door. It's nothing urgent but thought I'd put my thoughts out there. Many thanks for the wonderful NVR.

**Comment by chriswood1001** [comment]
I second this. I either want interior cameras where birdseye presents where my children are playing, OR want night time security views when motion has been detected. As of right now, watching my kids play is also mixed with cars driving by in the front yard.

**Reply by hawkeye217** [reply]
The upcoming version of Frigate (0.14) allows you to create camera groups for customized live views. This isn't "dual birdseye", but it does allow you to group cameras according to what you're looking for.
